tests/test_proxy_api.py

```diff
@@ -107,8 +107,11 @@ def test_get_empower_projects(self):
 
         empower_projects_list = self.handler.GetEmpowerProjects()
         assert isinstance(empower_projects_list, list)
-        assert empower_projects_list[0] == "2021"
-        assert empower_projects_list[1] == "2021\\LI0539"
+        assert isinstance(empower_projects_list[0], dict)
+        assert "projectName" in empower_projects_list[0]
+        assert "shortName" in empower_projects_list[0]
+        assert empower_projects_list[0]["projectName"] == "2021"
```

Comment:
It would be better to use an example where the porjectName and shortName are different, to make sure we get the right one. The current test would pass if projectName was returned for both.

We also shouldn't care about the order of the list, since we can't be sure of what order the API serves it in. But that is find for now.

Reply:
I modified the tests with different names. Also testing 2 different items in the list  instead of just one. 

The order of the list is a good suggestion and going forward perhaps we can consider using natsort or some other sorting algorithm to reorder the list before return? This will help web app development, since the enduser then can expect the same output order in the drop down every time.